### DISCUSSION TITLE
Had to pad my activation-bytes with a leading 0, then it worked!

### DISCUSSION THREAD

**Discussion by ghost** [mkb79/audible-cli#45] [General]
Just in case this can help someone else. I was getting a 7 character activation-bytes, which was causing errors for AAXtoMP3. I added a leading 0 to make it 8 characters and then it worked. I don't know what the precise issue/solution definition is/was, but just glad it worked. Thanks!

**Comment by mkb79** [comment]
Thank you for this information. Will fix this in my `audible` package.

**Reply by ghost** [reply]
Thank you so much for this awesome tool! It saved me from having to install audible-activator and it's prerequisite packages which wouldn't be very easy using LFS distro.
